DeepProjection/pdbGenerator.py

```diff
@@ -36,36 +36,176 @@ def get_random_pdb(self):
                 pass
 
     
-    def download_all_existing_pdb(self):
-        # Downloads all PDB files from the Protein Data Bank
+    def download_pdb_in_range(self, lower, upper, dir):
```

Comment:
May be useful to assign default values: upper=None and dir=None
If upper is None, then just use lower to determine what to download.
If dir is None, download in current working directory